### PR TITLE
chore(release): 0.1.0

### PR DESCRIPTION
**Pull request by TarikGul**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+
+What is not supported:
+
+- ParaToPara
+- ParaToRelay
+- NFTs
+
+What is supported:
+
+- SystemToPara (ForeignAssets, LiquidPoolAssets, CrossChain Transfers)
```
Comment (IkerAlus) [inline CHANGELOG.md:17]:
```suggestion
- SystemToPara (native assets, foreign assets, liquid pool assets, CrossChain Transfers)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+- SystemToRelay (CrossChain Transfers)
```
Comment (IkerAlus) [inline CHANGELOG.md:18]:
```suggestion
- SystemToRelay (DOT, CrossChain Transfers)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+- RelayToParachain (CrossChain Transfers)
```
Comment (IkerAlus) [inline CHANGELOG.md:19]:
```suggestion
- RelayToParachain (DOT, CrossChain Transfers)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+- RelayToSystem (CrossChain Transfers)
```
Comment (IkerAlus) [inline CHANGELOG.md:20]:
```suggestion
- RelayToSystem (DOT, CrossChain Transfers)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+- SystemToSystem (ForeignAssets, LiquidPoolAssets, CrossChain Transfers)
```
Comment (IkerAlus) [inline CHANGELOG.md:21]: I think at the moment we only allow DOT transfers to other system

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,39 @@
...
+- ParaToSystem (ForeignAssets, CrossChain Transfers via either Xtokens, or polkadotXcm pallet).
```
Comment (IkerAlus) [inline CHANGELOG.md:22]:
```suggestion
- ParaToSystem (Asset Hub assets, foreign assets, CrossChain Transfers via either Xtokens, or polkadotXcm pallet).
```

**Review by IkerAlus — CHANGES_REQUESTED** [review]
Some suggestions around the functionalities of each direction.

The rest looks good to go.

File: CHANGELOG.md (hunk 2 above)
Comment (TarikGul) [reply CHANGELOG.md:18]: For this should we say `native relay token` instead of DOT?

File: CHANGELOG.md (hunk 5 above)
Comment (TarikGul) [reply CHANGELOG.md:21]: Sounds good, the support and logic is there when it becomes available!

File: CHANGELOG.md (hunk 2 above)
Comment (IkerAlus) [reply CHANGELOG.md:18]: good point! yes, forevery occurrence of DOT that I added

**Review by IkerAlus — APPROVED** [review]
LGTM!

**Pull request merged**